.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,13 @@
+## Description
+
+
+## Steps For Releasing New Gem Version
+- Ensure all desired changes are merged to the `main` branch
```

Comment:
I dig it. Awesome. If you open this up locally to make the other change, it'd be cool to make these checkboxes, but it's good as is.

Reply:
Updated with checkboxes instead! 🙌 

![Screenshot 2022-10-23 at 14 41 22](https://user-images.githubusercontent.com/20643587/197395614-30fd9239-e942-48aa-849f-49162548a63e.png)